test/core/derivatives/convolution_settings.py

```diff
@@ -24,26 +24,65 @@
 ###############################################################################
 
 example = {
-    "module_fn": lambda: torch.nn.Conv2d(
-        in_channels=2,
-        out_channels=3,
+    "module_fn": lambda: torch.nn.Conv1d(
+        in_channels=4,
+        out_channels=6,
         kernel_size=2,
-        bias=False,
-        padding=1,
-        stride=2,
-        dilation=2,
+        groups=2,
     ),
-    "input_fn": lambda: torch.rand(size=(3, 2, 7, 7)),
-    "device": [torch.device("cpu")],  # optional
-    "seed": 0,  # optional
-    "id_prefix": "conv-example",  # optional
+    "input_fn": lambda: torch.rand(size=(3, 4, 7)),
+    "id_prefix": "groups-2",
 }
 CONVOLUTION_SETTINGS.append(example)
-
+# example = {
+#     "module_fn": lambda: torch.nn.Conv2d(
+#         in_channels=4,
+#         out_channels=6,
+#         kernel_size=2,
+#     ),
+#     "input_fn": lambda: torch.rand(size=(3, 4, 7, 7)),
+#     "device": [torch.device("cpu")],  # optional
+#     "seed": 0,  # optional
+#     "id_prefix": "conv-example",  # optional
+# }
+# CONVOLUTION_SETTINGS.append(example)
```

Comment:
Is this legacy from the debugging phase? I think it can be removed so there is no git diff

Reply:
removed